documentation/src/main/markdown/releasenotes.md

```diff
@@ -4,7 +4,8 @@
 
 ### Changed features
 
-* Signature scans, when invoked from the CLI, will now wait for all scans, such as snippet and string search, to complete before parsing Scan IDs.
+* Package Manager and Signature Scans will now query Black Duck directly when using the detect.wait.for.results property. This allows Detect to determine if results are ready much faster than waiting for a notification from Black Duck. Note: this feature requires Black Duck 2023.1.1 or later.
```

Comment:
Some minor edit suggestions, mostly replacing product names with the variables.

* Package Manager and Signature Scans will now query [blackduck_product_name] directly when using the detect.wait.for.results property. This expedites scanning by allowing [solution_name] to determine if results are ready, rather than waiting for a notification from [blackduck_product_name].   
**Note:** this feature requires [blackduck_product_name] 2023.1.1 or later.
* When waiting for results, Signature Scans will now wait for all scans that the Signature Scan could invoke, such as Snippet and String Search scans. Previously, only the Signature Scan itself was checked for completion.   
**Note:** this improvement requires [blackduck_product_name] 2023.1.2 or later.